<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ex5_mapNaviClass</class>
 <widget class="QMainWindow" name="ex5_mapNaviClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1045</width>
    <height>790</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1045</width>
    <height>790</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>公交转乘</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGraphicsView" name="graphicsView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>1045</width>
        <height>790</height>
       </size>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/ex5_mapNavi/华中科技大学周边地图.bmp);</string>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustIgnored</enum>
      </property>
      <property name="resizeAnchor">
       <enum>QGraphicsView::AnchorViewCenter</enum>
      </property>
      <property name="viewportUpdateMode">
       <enum>QGraphicsView::MinimalViewportUpdate</enum>
      </property>
      <property name="rubberBandSelectionMode">
       <enum>Qt::IntersectsItemShape</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1045</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件(F)</string>
    </property>
    <addaction name="action_open"/>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menu_Q">
    <property name="title">
     <string>查询(Q)</string>
    </property>
    <addaction name="action_zszc"/>
    <addaction name="action_zdjl"/>
    <addaction name="action_searchLoc"/>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string/>
    </property>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string/>
    </property>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string/>
    </property>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_Q"/>
   <addaction name="menu_H"/>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_open">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>读入地图</string>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
  </action>
  <action name="action_zszc">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>最少转乘</string>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
  </action>
  <action name="action_zdjl">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>最短距离</string>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
  </action>
  <action name="action_cxbz">
   <property name="text">
    <string/>
   </property>
  </action>
  <action name="action_gy">
   <property name="text">
    <string/>
   </property>
  </action>
  <action name="action_exit">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
  </action>
  <action name="action_searchLoc">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>查询地图点</string>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ex5_mapNavi.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>action_exit</sender>
   <signal>triggered(bool)</signal>
   <receiver>ex5_mapNaviClass</receiver>
   <slot>closewnd()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>524</x>
     <y>383</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_open</sender>
   <signal>triggered(bool)</signal>
   <receiver>ex5_mapNaviClass</receiver>
   <slot>loadmap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>524</x>
     <y>383</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_zszc</sender>
   <signal>triggered(bool)</signal>
   <receiver>ex5_mapNaviClass</receiver>
   <slot>zszc()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>522</x>
     <y>394</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_zdjl</sender>
   <signal>triggered(bool)</signal>
   <receiver>ex5_mapNaviClass</receiver>
   <slot>zdjl()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>522</x>
     <y>394</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_searchLoc</sender>
   <signal>triggered(bool)</signal>
   <receiver>ex5_mapNaviClass</receiver>
   <slot>searchLoc()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>522</x>
     <y>394</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>loadmap()</slot>
  <slot>closewnd()</slot>
  <slot>zszc()</slot>
  <slot>zdjl()</slot>
  <slot>searchLoc()</slot>
 </slots>
</ui>
